using-list-and-join.py

```diff
@@ -0,0 +1,4 @@
+words = ["Hello", "World"]
+
+message = " ".join(words)
+print(message)
```

Comment:
There is an existing implementation for this solution that you've submitted!

You can find the existing solution [here](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-join.py). Using the `.join()` method to join the elements of a list has been already implemented. I'll need you to remove this implementation from this PR as well.